api/task_check_run.go

```diff
@@ -73,6 +73,8 @@ const (
 	TaskCheckDatabaseStatementSyntax TaskCheckType = "bb.task-check.database.statement.syntax"
 	// TaskCheckDatabaseStatementCompatibility is the task check type for statement compatibility.
 	TaskCheckDatabaseStatementCompatibility TaskCheckType = "bb.task-check.database.statement.compatibility"
+	// TaskCheckDatabaseStatementRequireWhere is the task check type for the WHRER clause requirement for UPDATE/DELETE.
+	TaskCheckDatabaseStatementRequireWhere TaskCheckType = "bb.task-check.database.statement.require-where"
```

Comment:
bb.task-check.database.statement.where.require

we will have [where.no-leading-wildcard-like](https://bytebase.com/database-review-guide#query-where-no-leading-wildcard-like) later

Reply:
Do we need to distinguish between `query requires WHERE` and `DML requires WHERE`?
If `bb.task-check.database.statement.where.require`, I think we treat them as a single rule.
Is there something wrong with my understanding?

Reply:
We treat them as a single rule. My feeling is it's better to enforce WHERE regardless of the query type.

Reply:
Yeah, this rule will be used for DML only. For query types, users should use our SQL editor.